plugins/phone-ringer/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+find_packages(PHONE_RINGER_PACKAGES REQUIRED
+    Canberra
+    Gee
+    GLib
+    GModule
+    GObject
+    GDKPixbuf2
+)
+
+vala_precompile(PHONE_RINGER_VALA_C
+SOURCES
+    src/plugin.vala
+    src/register_plugin.vala
+CUSTOM_VAPIS
+    ${CMAKE_BINARY_DIR}/exports/xmpp-vala.vapi
+    ${CMAKE_BINARY_DIR}/exports/dino.vapi
+    ${CMAKE_BINARY_DIR}/exports/qlite.vapi
```

Comment:
You are not using xmpp-vala or qlite in this plugin.

Reply:
When I remove:
```
${CMAKE_BINARY_DIR}/exports/xmpp-vala.vapi
${CMAKE_BINARY_DIR}/exports/qlite.vapi
```
I get a whole bunch of this when running make:
```
...build/exports/dino.vapi:1289.80-1289.83: error: The symbol `Xmpp' could not be found
...build/exports/dino.vapi:1273.60-1273.67: error: The symbol `Xep' could not be found
...build/exports/dino.vapi:774.11-774.15: error: The symbol `Qlite' could not be found
```

I looked around a bit in other CMakeList.txt files. It seems it is needed to have those three together? I am not a CMake wizard, do I have to do more things then just remove the lines to get it to compile ?

Reply:
You're right, those need to stay, my bad.